### ISSUE TITLE
unmappable character for encoding UTF8

### ISSUE THREAD

**Issue by gugica** [amzn/alexa-avs-raspberry-pi#133]
When running command mvn -e -X install i get error messages about unmappable characters for encoding UTF8

[ERROR] /home/pi/Documents/software/alexa-avs-raspberry-pi-master/samples/javaclient/src/main/java/com/amazon/alexa/avs/message/.AppleDouble/DialogRequestIdHeader.java:[2,8] unmappable character for encoding UTF8
[ERROR] /home/pi/Documents/software/alexa-avs-raspberry-pi-master/samples/javaclient/src/main/java/com/amazon/alexa/avs/message/.AppleDouble/DialogRequestIdHeader.java:[2,20] unmappable character for encoding UTF8
[ERROR] /home/pi/Documents/software/alexa-avs-raspberry-pi-master/samples/javaclient/src/main/java/com/amazon/alexa/avs/message/.AppleDouble/DialogRequestIdHeader.java:[2,32] unmappable character for encoding UTF8
[ERROR] /home/pi/Documents/software/alexa-avs-raspberry-pi-master/samples/javaclient/src/main/java/com/amazon/alexa/avs/message/.AppleDouble/DialogRequestIdHeader.java:[2,60] unmappable character for encoding UTF8
[ERROR] /home/pi/Documents/software/alexa-avs-raspberry-pi-master/samples/javaclient/src/main/java/com/amazon/alexa/avs/message/.AppleDouble/DialogRequestIdHeader.java:[2,72] unmappable character for encoding UTF8

And many more lines reporting the same problems. Under that i also get error messages about illegal characters:

[ERROR] /home/pi/Documents/software/alexa-avs-raspberry-pi-master/samples/javaclient/src/main/java/com/amazon/alexa/avs/message/.AppleDouble/DialogRequestIdHeader.java:[2,42] illegal character: '\u0000'
[ERROR] /home/pi/Documents/software/alexa-avs-raspberry-pi-master/samples/javaclient/src/main/java/com/amazon/alexa/avs/message/.AppleDouble/DialogRequestIdHeader.java:[2,43] illegal character: '\u0002'
[ERROR] /home/pi/Documents/software/alexa-avs-raspberry-pi-master/samples/javaclient/src/main/java/com/amazon/alexa/avs/message/.AppleDouble/DialogRequestIdHeader.java:[2,44] class, interface, or enum expected
[ERROR] /home/pi/Documents/software/alexa-avs-raspberry-pi-master/samples/javaclient/src/main/java/com/amazon/alexa/avs/message/.AppleDouble/DialogRequestIdHeader.java:[2,45] illegal character: '\u0000'
[ERROR] /home/pi/Documents/software/alexa-avs-raspberry-pi-master/samples/javaclient/src/main/java/com/amazon/alexa/avs/message/.AppleDouble/DialogRequestIdHeader.java:[2,46] illegal character: '\u0000'

I would be very happy is someone could post a solution to this problem.

**Comment by xozai** [comment]
I have the same error.

**Comment by mopby** [comment]
same here. Raspberry Pi 3 with newest Rasbian Jessie (march 18)

**Comment by keleric** [comment]
I don't have the .AppleDouble folder on my machine. Try 'rm -rf samples/javaclient/src/main/java/com/amazon/alexa/avs/message/.AppleDouble', then mvn clean && mvn install && mvn exec:exec

**Comment by gugica** [comment]
I am still getting the same error message. Even if I delete .AppleDouble, the folder is still mentioned in error report.

**Comment by keleric** [comment]
Are you still having this issue?

**Comment by gugica** [comment]
Still no luck. However I noticed this line which comes right before installation error report:

Downloaded: https://repo.maven.apache.org/maven2/com/google/collections/google-collections/1.0/google-collections-1.0.jar (625 KB at 166.7 KB/sec)
[INFO] Changes detected - recompiling the module!
[WARNING] File encoding has not been set, using platform encoding UTF-8, i.e. build is platform dependent!
[INFO] Compiling 250 source files to /home/pi/Documents/software/Alexa/samples/javaclient/target/classes
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /home/pi/Documents/software/Alexa/samples/javaclient/src/main/java/com/amazon/alexa/avs/.AppleDouble/SpeakItem.java:[2,8] unmappable character for encoding UTF8

**Comment by elstryan** [comment]
In the past I've seen this error when there are incorrectly encoded characters in your config.json file. 

Can you do a few things: 
1) Verify you config.json file against this one: https://github.com/rhee/alexa-avs-raspberry-pi/commit/3f5a360c3af5014b1d1f32786e52fbac8ae19903 
2) How did you edit you config.json? 
3) Can you remove the pass phrase and post your config file so that we can download it and make sure that it looks correct.

**Comment by ajotwani** [comment]
We're closing this Issue since the original poster (or another commenter) hasn't yet responded to the question or request made to them 14 days ago. We therefore assume that the user has lost interest or resolved the problem on their own.

Don't worry though; if this is an error, let us know with a comment and we'll be happy to reopen the Issue.

Thanks!

**Comment by gugica** [comment]
Sorry. I wasn't checking my email for a while so I missed updates on this issue.
So I tried to make a clean install of alexa with reinstalling OS. I noticed that Alexa installation guide is now much more detailed and simplified and sets up much more things instead of you.

However Issue is still present and the error message did not change.
I also checked the config.json as you said but I couldn't find anything unusual.
![screen shot 2016-08-02 at 14 08 58](https://cloud.githubusercontent.com/assets/18504403/17328001/77e4e21a-58bb-11e6-926b-bfdd27a3d7b5.png)

Anyway I am grateful for your support. Im looking forward to get this thing finally working.
